stable/tomcat/templates/appsrv.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "tomcat.fullname" . }}
+  labels:
+    app: {{ template "tomcat.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+      metadata:
+       labels:
+        app: {{ template "tomcat.name" . }}
+        release: {{ .Release.Name }}
+      spec:
+       volumes:
+       - name: app-volume
+         emptyDir: {}
+       containers:
+       - name: war
+         image: {{ .Values.image.webarchive.repository }}:{{ .Values.image.webarchive.tag }}
+         imagePullPolicy: {{ .Values.image.pullPolicy }}
+         volumeMounts:
+         - name: app-volume
+           mountPath: /app
+       - name: {{ .Chart.Name }}
+         image: {{ .Values.image.tomcat.repository }}:{{ .Values.image.tomcat.tag }}
+         imagePullPolicy: {{ .Values.image.pullPolicy }}
+         command: ["sh","-c", {{ .Values.cmd.start }}]
+         volumeMounts:
+         - name: app-volume
+           mountPath: {{ .Values.deploy.directory }}
+         ports:
+         - containerPort: {{ .Values.service.internalPort }}
+           hostPort: 8009
+         livenessProbe:
+           tcpSocket:
+             port: 8080
+           initialDelaySeconds: 15
+           periodSeconds: 20  
+         readinessProbe:
+           httpGet:
+             path: /sample
```

Comment:
Sample? This should be an endpoint in the deployed app.

Reply:
I removed that. I tight back to the instance readiness. /sample is the app to be ready. 

Thanks for all the feedback and sorry for being so novice here. I learn a lot about how to be disciplined with charts as this is one of the reasons we want to use stable/charts as the source of truth.